### ISSUE TITLE
quickfix output doesn't work in windows 10

### ISSUE THREAD

**Issue by kzmath** [pianocomposer321/yabs.nvim#25]
When I use output = "quickfix", I got the following error:

Error executing  lua <USERDIR>\.vim\plugged\yabs.nvim\lua\yabs\util.lua:58: bad argument #1 to 'new_pipe' (boolean expected, got no value)

I am using this config:
python = {
	tasks = {
		run = {
			command = "dtx-env\\Scripts\\python %",
			output = "quickfix",
			} 
		},
	},
on neovim 5.1 on windows 10.

My guess that this is related to windows, hence the title.

P.S. Great plugin! I am using it even without quickfix.

**Comment by JSchrtke** [comment]
> My guess that this is related to windows, hence the title.
Negative, I just got this on ubuntu 20.04 as well.

**Comment by pianocomposer321** [comment]
@JSchrtke can you provide a bit more context as to what you were doing when you got this error (i.e. steps to reproduce)?

**Comment by JSchrtke** [comment]
Ye, sorry, I was feeling lazy :P

Give me a couple minutes, I'll give you a minimal config

**Comment by pianocomposer321** [comment]
@Shatur would you mind checking this out?

@JSchrtke and @kzmath just to make sure, what version of neovim are you running (what's the output of `nvim --version`)?

**Comment by JSchrtke** [comment]
Alright, here we go.

`Ubuntu 20.04`

```
NVIM v0.5.2-dev+20-g1e1fd71fe
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/js/develop/c/neovim/build/config -I/home/js/develop/c/neovim/src -I/home/js/develop/c/neovim/.deps/usr/include -I/usr/include -I/home/js/develop/c/neovim/build/src/nvim/auto -I/home/js/develop/c/neovim/build/include
Compiled by root@js-ubuntu

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/js/local/nvim/share/nvim"

Run :checkhealth for more info
```
in 
```
kitty 0.20.3 created by Kovid Goyal
```
and 
```
fish, version 3.3.1
```
latest version of `yabs.nvim`

vim config
```vim
call plug#begin('~/.vim/plugged')
Plug 'pianocomposer321/yabs.nvim'
call plug#end()

lua << EOF
require('yabs'):setup({
    languages = {
        go = {
            tasks = {
                build = {
                    command = 'go build .',
                    output = 'quickfix',
                }
            }
        }
    }
})
EOF
```

I used the following `main.go` file

```go
package main

import "fmt"

func main() {
    fmt.Println("Hello, World!")
}
```

and ran `lua require('yabs'):run_task('build')`

which yielded the following error

```
E5108: Error executing lua /home/js/.vim/plugged/yabs.nvim/lua/yabs/utils.lua:66: bad argument #1 to 'new_pipe' (boolean expected, got no value)
```

**Comment by pianocomposer321** [comment]
@JSchrtke Neovim 0.5._2_? I didn't know that that existed...my guess is it only did for a short time as a nightly build? When's the last time you updated your nightly?

Unfortunately, I'm unable to test this right now (which is why I pinged @Shatur), so the only thing I can suggest is that you try it on the latest stable version, which is 0.6.0 right now. It's possible that this is a bug in your version of neovim nightly.

**Comment by JSchrtke** [comment]
hm, so, 
```
NVIM v0.6.1-dev+653-ga4699892a
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by root@js-ubuntu

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/js/local/nvim/share/nvim"

Run :checkhealth for more info
```

doesn't error out

```vim
NVIM v0.5.2-dev+20-g1e1fd71fe
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/js/develop/c/neovim/build/config -I/home/js/develop/c/neovim/src -I/home/js/develop/c/neovim/.deps/usr/include -I/usr/include -I/home/js/develop/c/neovim/build/src/nvim/auto -I/home/js/develop/c/neovim/build/include
Compiled by root@js-ubuntu

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/js/local/nvim/share/nvim"

Run :checkhealth for more info
```
still does. That one is the latest version of the `release-0.5` branch. There were some backports of vim patches that happened after the `0.6` release.

**Comment by Shatur** [comment]
> @Shatur would you mind checking this out?

Will take a look!

**Comment by Shatur** [comment]
Can someone check if #37 fixes the issue?

**Comment by JSchrtke** [comment]
I'll give this a go tonight. I'd rather not break my nicely working config on my work machine ;)

**Comment by pianocomposer321** [comment]
@JSchrtke #37 is ready to merge. Did you get a chance to see if it fixes the problem? We'd like your feedback before we merge.

**Comment by pianocomposer321** [comment]
BTW, despite the name, it's not actually a breaking change config-wise, it's only breaking in that plenary is now a dependency.

**Comment by kzmath** [comment]
Just tested, #37  fixed my problem (windows 10, neovim 0.6.0). Thanks!

**Comment by JSchrtke** [comment]
Yeah, sorry for the absence, I didn't get around to testing it until now. Works fine now on my Ubuntu 20.04 as well. Thanks guys!
